design/ceph/local-node-agent.md

```diff
@@ -192,11 +192,12 @@ When a Rook agent receives a cluster CRD delete event, they will respond by chec
 
 To forcefully remove the storage from local pods, the agent will perform the following sequence of steps for each Rook PVC:
 ```bash
-$ kubectl delete pvc <pvc name>
-$ sudo rbd unmap -o force /dev/rbdX
+kubectl delete pvc <pvc name>
```

Comment:
What if we use `$var` syntax instead of `<var>` so that the whole var gets highlighted?

```suggestion
kubectl delete pvc $pvc_name
```